www/docs/en/dev/guide/overview/project-structure.md

```diff
@@ -0,0 +1,105 @@
+---
+license: >
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+title: Project Structure of a Cordova Project
+toc_title: Project Structure
+description: 
+---
+
+# Project Structure
+
+## CLI project directory structure
+
+Cordova CLI works with the following directory structure:
+
+```
+myapp/
+|-- package.json
+|-- config.xml
+|-- hooks/
+|-- merges/
```

Comment:
The merges folder is not created automatically anymore, and we kinda discourage its use (although it's still supported as far as I know).

Also, the example folders here are indented one level too far.

Reply:
Is what is written below about `merges` still accurate though?
Will have to splti existing document into "created by default" and "also supported" somehow I guess.

Reply:
I applied changes so this can be re-reviewed.

I tried to reorg the structure so there is now the "default" and "optional" section.